ga4gh/datamodel/__init__.py

```diff
@@ -229,6 +256,8 @@ class VariantSetCompoundId(DatasetCompoundId):
     """
     fields = DatasetCompoundId.fields + ['variantSet']
     containerIds = DatasetCompoundId.containerIds + [('variantSetId', 1)]
+    obfuscators = [DatasetCompoundId, None]
```

Comment:
@dcolligan, what is the rationale for obfuscating the individual components rather
than the id as a whole.

Danny Colligan <notifications@github.com> writes:
> In ga4gh/datamodel/__init__.py:
> 
> > @@ -229,6 +256,8 @@ class VariantSetCompoundId(DatasetCompoundId):
> >      """
> >      fields = DatasetCompoundId.fields + ['variantSet']
> >      containerIds = DatasetCompoundId.containerIds + [('variantSetId', 1)]
> > +    obfuscators = [DatasetCompoundId, None]
> 
> We need a list of obfuscators because in the CompoundId's __init__ method,
> obfuscate is called on every component part of the id to set an attribute in
> the current compoundId.
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub*
>